core/server/proxy/src/main/java/alluxio/proxy/s3/S3RestServiceHandler.java

```diff
@@ -498,6 +514,8 @@ public Response createBucket(@PathParam("bucket") final String bucket,
             SetAttributePOptions attrPOptions = SetAttributePOptions.newBuilder()
                 .putAllXattr(xattrMap)
                 .setXattrUpdateStrategy(File.XAttrUpdateStrategy.UNION_REPLACE)
+                .setCommonOptions(
```

Comment:
I think it's not needed here. we must have the bucket, then we can put the tagging to the bucket.